### DISCUSSION TITLE
[Feature] Open Image From "n years since..."

### DISCUSSION THREAD

**Discussion by ghost** [immich-app/immich#5225] [Feature Request]
### The feature

I love the "*n* years since..." feature, but it would be **perfect** if I could actually load the full image from one of the image previews.

Right now tapping just takes you to the next image, which is fine; but sometimes I want to see the full image without having to dig through 100s or 1000s of photos to find it. :)

I'm imagining maybe a long press or something to get a "load full image" or something?

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by kamzil** [comment]
EDIT: Looks like I was kind of missing something. In the web version when you open a memory, you can actually scroll down and click any of the photos in the grid, which opens a fully-featured modal with Details, album actions etc. That said, it would still be handy to have these controls already present in the memory when you open it from the timeline, so that you don't need to go through the memories grid to find it. Or alternatively being able to go there by clicking the photo.

Old message:
I was thinking that I'm missing something because I kept trying to get to the actual photo from the "years since" view. It would feel like a very natural thing to be able to do with this kind of feature. Are there some technical limitations why this has not been implemented that way?

**Reply by oledid** [reply]
I want to be able to navigate to an image from "memories" so that I can add it to a special album, delete it, archive it etc. Or just see or update its metadata.

Could not find a way to do this today on iPhone, other than looking up the image manually in the timeline.

**Reply by fbloemhof** [reply]
I have the same wish as @oledid, it would be great to navigate to the actual foto out of the memory.

I'm an Android user.
